src/core/qgsvectorfilewriter.h

```diff
@@ -555,6 +556,25 @@ class CORE_EXPORT QgsVectorFileWriter : public QgsFeatureSink
          * \since QGIS 3.34
          */
         bool includeConstraints = false;
+
+        /**
+         * Set to TRUE to transfer field domains to the exported vector file.
+         *
+         * Support for field domains depends on the output file format.
```

Comment:
"\note Only available in builds based on GDAL 3.5 or later"

Reply:
done